fs/io_zenfs.cc

```diff
@@ -322,12 +327,15 @@ IOStatus ZoneFile::PositionedRead(uint64_t offset, size_t n, Slice* result,
 
   r_off = 0;
   extent = GetExtent(offset, &r_off);
```

Comment:
The extent is being read in GetExtent, and the ReadLock is not protecting that. so this does not work.